packages/@okta/vuepress-theme-prose/assets/css/okta/pages/multi-tenancy.scss

```diff
@@ -0,0 +1,45 @@
+.multi-tenancy {
```

Comment:
I don't think that styles are really necessary. Tested without them, haven't seen much difference. But it's my opinion.

Reply:
Thanks for your feedback Andrii. There were additional styles in the css that were removed pending a design review. Just to get the first version of this document out, I tried to keep the css has minimal as possible.  There will be additional versions of this document in the proceeding weeks which may include more styles.

Reply:
Also, in the latest commit, the remaining css is focused on resizing some of the images in the document.